packages/graphql-server-module-graphiql/src/renderGraphiQL.ts

```diff
@@ -66,6 +68,7 @@ export function renderGraphiQL(data: GraphiQLData): string {
   <script src="//cdn.jsdelivr.net/react/15.0.0/react.min.js"></script>
   <script src="//cdn.jsdelivr.net/react/15.0.0/react-dom.min.js"></script>
   <script src="//cdn.jsdelivr.net/graphiql/${GRAPHIQL_VERSION}/graphiql.min.js"></script>
+  <script src="//unpkg.com/subscriptions-transport-ws@0.5.5/browser/client.js"></script>
```

Comment:
Why do we need to pull the package when the user didnt provide the websocket endpoint?